### PR TITLE
Mention the Go version in the README

### PR DESCRIPTION
**Pull request by arianvp**
Ubuntu 16.04 ships with go `1.6.2` which does not have the `"context"` package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ HTTP, HTTPS, DNS, TCP and ICMP.
 ## Building and running
 
 ### Local Build
+First of all, you need at least Go version `1.7`
```
Comment (brian-brazil) [inline README.md]: I'd suggest using the same text in Prometheus itself's readme

File: README.md [hunk 2]
```diff
@@ -10,7 +10,10 @@ HTTP, HTTPS, DNS, TCP and ICMP.
 ## Building and running
 
 ### Local Build
-First of all, you need at least Go version `1.7`
+Building from source
+
+To build Prometheus from the source code yourself you need to have a working
```
Comment (brian-brazil) [inline README.md]: s/Prometheus/the blackbox exporter/

**Comment by sofianinho** [comment]
Hi @brian-brazil would something like my recent PR be useful for this case ?

**Comment by brian-brazil** [comment]
This is stale, and we have published builds.

**Pull request closed without merging**